.github/workflows/ci_contracts.yml

```diff
@@ -0,0 +1,24 @@
+name: contracts
+
+on: [push, pull_request]
```

Comment:
Not a big deal, but we can just run `push` against master to avoid duplicate runs.

Reply:
Done :)